### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by shihabrafio**
Create a database named vet_clinic.
Create a table named animals with the following columns:
- id: integer
- name: string
- date_of_birth: date
- escape_attempts: integer
- neutered: boolean
- weight_kg: decimal

- Insert the following data:
  
   - Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
   - Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
   - Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
   - Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
   
- Write queries for the following quests:
  - Find all animals whose name ends in "mon".
    
 image

  - List the name of all animals born between 2016 and 2019.
   
image

  - List the name of all animals that are neutered and have less than 3 escape attempts.
  
image

  - List the date of birth of all animals named either "Agumon" or "Pikachu".
  
image

  - List name and escape attempts of animals that weigh more than 10.5kg
  
image

  - Find all animals that are neutered.
  
image

  - Find all animals not named Gabumon.
  
image

  - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
  
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <a name="readme-top"></a>
 <div align="center">
 
-  <h3><b>Microverse README Template</b></h3>
+  <h3><b>Vet Clinic</b></h3>
 
 </div>
```
Comment (Gambit142) [inline README.md:7]: - [ ] Kindly include a valid LICENSE link to the LICENSE section in your README file. When I click the link, it displays a 404 error page. Adding an open-source license to your repository ensures that others can use, copy, modify, and contribute back to your project. It's an important step when creating an open-source project. 😎

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @shihabrafio 👋, 

Good job so far!

## Highlights
- Everything that creates or changes the structure of the table is in the schema.sql file.
- Everything that populates tables with data is in the data.sql file.
- Everything that is a query that helps to analyze data is in the queries.sql file.

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by binhussen — APPROVED** [review]
Hi @shihabrafio , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Created schema, data, and queries ✔️
- Well Structured Query ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by shella12 — COMMENTED** [review]
Hi @shihabrafio, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Wrote required queries ✔️ 
- Created animal table ✔️ 
- Inserted required data ✔️ 
- Implemented all requirements 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**